test/e2e/stats_test.go

```diff
@@ -75,6 +75,15 @@ var _ = Describe("Podman stats", func() {
 	})
 
 	It("podman stats only output cids", func() {
+		session := podmanTest.RunTopContainer("")
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		session = podmanTest.Podman([]string{"stats", "--all", "--no-stream", "--format", "\"{{.ID}} {{.UpTime}} {{.AVGCPU}}\""})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
```

Comment:
Can you look for the header in the the output?

Reply:
@rhatdan do you mean something like

 `session = podmanTest.Podman([]string{"stats", "--all", "--no-stream", "--format", "\"{{.ID}} {{.UpTime}} {{.AVGCPU}}\""})`
`out := session.OutputToStringArray()`
`Expect(len(out)).To(BeNumerically("==", 3))`

or would that not work? It seems that none of the stats tests actually validate the output too much.

Reply:
No I just want you to make sure the headings you expect show up in the output.
	Expect(session.LineInOutputContains("UpTime").To(BeTrue())
	Expect(session.LineInOutputContains("AVGCPU").To(BeTrue())